test/CMakeLists.txt

```diff
@@ -301,6 +301,10 @@ if( iodaconv_bufr_query_ENABLED )
     testinput/bufr_query_fieldname_validation.py
     testinput/bufr_ncep_rtma_mesonet.yaml
     testinput/bufr_ncep_rtma_adpsfc.yaml
+    testinput/rtma_ru.t0000z.aircft.tm00_nc004006.bufr_d
```

Comment:
Is an rtma aircraft file different than a regular aircraft file?

Reply:
@nicholasesposito thanks, there are separate bufr files for global and regional data. BUFR table is same for both, but size and variables (mnemonics) vary.

Reply:
Understood. Thanks

Reply:
tested on those two files it worked successfully for the `aircft` file

Reply:
> are the regional aircraft files available in NOMADS? I don't even see `aircraft` but see `airsev`
> 
> on the NCAR RDA ,, NOAA provides files like `gdas.aircar.t00z.20230317.bufr` and then the `ds351.0/bufr/2023/gdas.aircft.t00z.20230317.bufr` do you know if the `aircar` are the regional ones?

@BenjaminRuston thanks, I don't know if regional aircraft files are available in NOMADS. But aircar are also regional ones.